v1/lib/static/css/main.css

```diff
@@ -2354,3 +2354,57 @@ input::placeholder {
   padding: 5px 0;
 }
 /* End of Footer */
+
+
+.tabs {
+  border-top: 1px solid #cfcfcf;
+}
+
+.nav-tabs {
+  display: flex;
+  border-bottom: 4px solid #e0e0e0;
+  width: 100%;
+  padding: 0;
+  overflow-x: auto;
+  white-space: nowrap;
+  max-height: 100%;
+}
+
+.nav-tabs::-webkit-scrollbar {
+  display: none;  // Safari and Chrome
```

Comment:
I guess i should not put comment on this :/

Reply:
In CSS comments should be `/* */` not  `//`